tests/test_utils.py

```diff
@@ -4,7 +4,7 @@
 
 import numpy as np
 import pandas as pd
-
+import numpy.testing as npt
```

Comment:
```suggestion
from numpy.testing import assert_almost_equal
```

To be consistent with the other test files.